msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionTopicBean.java

```diff
@@ -556,6 +558,62 @@ public void setPostFirst(String postFirst)
       topic.setPostFirst(Boolean.valueOf(false));
     }
   }
+
+  /**
+   * @return Returns the postAnonymous status.
+   */
+  public String getPostAnonymous()
+  {
+    LOG.debug("getPostAnonymous()");
+    if ("".equals(postAnonymous))
+    {
+      boolean isAnonymous = topic != null && topic.getPostAnonymous() != null && topic.getPostAnonymous().booleanValue();
+      postAnonymous = Boolean.valueOf(isAnonymous).toString();
+    }
+    return postAnonymous;
+  }
+
+  /**
+   * @param postAnonymous
+   * Set the postAnonymous status
+   */
+  public void setPostAnonymous(String postAnonymous)
+  {
+    if (LOG.isDebugEnabled())
+    {
+      LOG.debug("setPostAnonymous(String " + postAnonymous + ")");
+    }
+    boolean isAnonymous = Boolean.TRUE.toString().equals(postAnonymous);
+    topic.setPostAnonymous(Boolean.valueOf(isAnonymous));
+  }
+
+  /**
+   * @return Returns the revealIDsToRoles status.
+   */
+  public String getRevealIDsToRoles()
+  {
+    LOG.debug("getRevealIDsToRoles()");
+    if ("".equals(revealIDsToRoles))
+    {
+      boolean isRevealIDsToRoles = topic != null && topic.getRevealIDsToRoles() != null && topic.getRevealIDsToRoles().booleanValue();
```

Comment:
I think Booleans are just unboxed automatically now so the calls to booleanValue() aren't needed.